python/ray/tune/trainable/trainable.py

```diff
@@ -502,6 +503,56 @@ def save(
 
         return checkpoint_dir
 
+    def _get_latest_available_checkpoint(self) -> Optional[str]:
+        latest_local_checkpoint = self._get_latest_local_available_checkpoint()
+        latest_remote_checkpoint = self._get_latest_remote_available_checkpoint()
+
+        if not latest_local_checkpoint:
+            return latest_remote_checkpoint
+        elif not latest_remote_checkpoint:
+            return latest_local_checkpoint
+
+        # Else, both are available
+        return max([latest_local_checkpoint, latest_remote_checkpoint])
```

Comment:
is this comparing the absolute path between local and remote checkpoints?

Reply:
shouldn't we compare the iteration number instead?

Reply:
It should be fine as we do `os.path.join(self._logdir, name)` in both cases!